### DISCUSSION TITLE
In "Content" Module, the "Add Field" will be hidden when adding columns.

### DISCUSSION THREAD

**Discussion by tuutoo** [directus/directus#16055] [Feature Requests]
### Describe the Bug

In "Content" Module, the "Add Field" will be hidden when adding columns.

1. When you have a few fields, the "Add Field" button is OK
<img width="1909" alt="image" src="https://user-images.githubusercontent.com/9068314/196733965-3375ad97-816e-4e0e-9102-7521718972e2.png">

2. After some more columns added to the view, now the "Add Field" is beyond the screen, also not have a horizontal scrollbar to get it back. The only way you can do is hide other columns and get it back.
![image](https://user-images.githubusercontent.com/9068314/196735186-30247c2d-0791-4834-9f58-fc33cce857f1.png)

Expect to have this "Add Field" as a column and fixed to the right? And have a horizontal scroll bar to view all other columns?

Enhacement:
Personally, I think it's better to show/hide columns directly within the dropdown of "Add Fields", instead of "Press Hide this Field" in each column and then Add it back in anther place (Within "Add Field").

### To Reproduce

Press "Add Field" to add more columns, and the "Add Field" button will be out of the screen.

### What version of Directus are you using?

9.18.1

### What version of Node.js are you using?

16.18.0

### What database are you using?

Postgres 13

### What browser are you using?

Chrome

### How are you deploying Directus?

locally

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
